src/Mod/Draft/draftgeoutils/general.py

```diff
@@ -148,32 +148,30 @@ def isAligned(edge, axis="x"):
 
     The axis can be 'x', 'y' or 'z'.
     """
+    def is_same(a, b):
+      return round(a, precision()) == round(b, precision())
```

Comment:
Any reason to not use something like ` return abs(a-b) < precision()` ?

Reply:
If I am not mistaken it would then have to be:
`return abs(a-b) < 10 ** (-precision() - 1)`

In other parts of the file `round` is used though. I believe you would call this 'local consistency'... But both solutions are fine by me.

Reply:
What I don't like about the `isAligned` function is the repetition of (almost) the same code block for x, y and z.